src/demo/DemoTestClientService.java

```diff
@@ -120,6 +122,43 @@ public static void main(String[] args) {
 		}
 	}
 
+	private static String testUpdateClientAssociations(Client client) throws MambuApiException {
+
+		String methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println("\nIn " + methodName);
+		
+		client.setAssignedBranchKey(DemoUtil.getDemoClient().getAssignedBranchKey());
+		client.setAssignedCentreKey(DemoUtil.getDemoClient().getAssignedCentreKey());
+		client.setAssignedUserKey(DemoUtil.getDemoClient().getAssignedUserKey());
+		
+		// null fields not wanted in patch
+		// and keep only the fields defining associations
+		client.setFirstName(null);
+		client.setLastName(null);
+		client.setMiddleName(null);
+		client.setEmailAddress(null);
+		client.setMobilePhone1(null);
+		client.setHomePhone(null);
+		client.setState(null);
+		client.setClientRole(null);
+		client.setId(null);
+		client.setPreferredLanguage(null);
+		client.setBirthDate(null);
+
+		ClientsService clientService = MambuAPIFactory.getClientService();
+		boolean status = clientService.patchClient(client);
+		System.out.println("Update status=" + status);
+
+		// Get updated client details back to confirm PATCHed values
+		Client updatedClient = clientService.getClient(client.getEncodedKey());
+		String updatedId = updatedClient.getId();
+		System.out.println("\tUpdate AssignedBranchKey=" + updatedClient.getAssignedBranchKey() + "\tAssignedCentreKey="
+				+ updatedClient.getAssignedCentreKey() + "\tAssignedUserKey=" + updatedClient.getAssignedUserKey());
+
+		return updatedId;

```

Comment:
The reason of returning the id is to use the same object in the future tests. Currently not used . I can change the signature of the method.